src/engine/shared/console.cpp

```diff
@@ -667,9 +668,52 @@ static void StrVariableCommand(IConsole::IResult *pResult, void *pUserData)
 		char aBuf[1024];
 		str_format(aBuf, sizeof(aBuf), "Value: %s", pData->m_pStr);
 		pData->m_pConsole->Print(IConsole::OUTPUT_LEVEL_STANDARD, "Console", aBuf);
+		str_copy(pResult->m_aValue, pData->m_pStr, sizeof(pResult->m_aValue));
 	}
 }
 
+void CConsole::Con_EvalIf(IResult *pResult, void *pUserData)
+{
+	CResult Result;
+	CConsole* pConsole = static_cast<CConsole *>(pUserData);
+	CCommand *pCommand = ((CConsole*)pUserData)->FindCommand(pResult->GetString(0), ((CConsole*)pUserData)->m_FlagMask);
+	char aBuf[128];
```

Comment:
Can be declared inside the following if since it's not used anywhere else.

Reply:
is this considered better style? Never heard of it.

Reply:
It makes it easier to see where a particular variable is used if it's declared in the smallest possible scope and as late as possible.

`Result` could also be declared after the if since it's only used then.